### PR TITLE
Adding idToken to support Raffle

### PR DESCRIPTION
**Pull request by mieng18**

### CODE REVIEW

File: Sources/Bitski-iOS-SDK/Core/Bitski.swift [hunk 1]
```diff
@@ -353,13 +361,14 @@ public class Bitski: NSObject, BitskiAuthDelegate {
...
+        authState.performAction { [weak self](accessToken, idToken, error) in
             guard let self = self else {
                 completion(nil, error)
                 return
             }
-            if let accessToken = accessToken {
+            if let accessToken = accessToken, let idToken = idToken {
                 self._authToken = accessToken
+                self._idToken = idToken
                 completion(accessToken, nil)
```
Comment (ptescher) [inline Sources/Bitski-iOS-SDK/Core/Bitski.swift:372]: How do we get the _idToken out? Just returning access token here.

Comment (mieng18) [reply Sources/Bitski-iOS-SDK/Core/Bitski.swift:372]: @ptescher     public var idToken: String {
       return _idToken
    } 

?

**Review by ptescher — APPROVED** [review]

File: Sources/Bitski-iOS-SDK/Core/Bitski.swift (hunk 1 above)
Comment (ptescher) [reply Sources/Bitski-iOS-SDK/Core/Bitski.swift:372]: so you call getCurrentAccessToken() first, then call .idToken ?

Comment (mieng18) [reply Sources/Bitski-iOS-SDK/Core/Bitski.swift:372]: yes _idToken will piggyback that call then the public variable will return the private _idToken I don't think it is needed for signing transactions. wdyt?

Comment (ptescher) [reply Sources/Bitski-iOS-SDK/Core/Bitski.swift:372]: fine for now

Comment (mieng18) [reply Sources/Bitski-iOS-SDK/Core/Bitski.swift]: @ptescher is it ok for now ?

**Pull request merged**